### PR TITLE
Fix constant refresh causes IE 10 to leak memory

### PR DESCRIPTION
**Pull request by zomaish**

### CODE REVIEW

File: src/captured_trace.js [hunk 1]
```diff
@@ -452,7 +452,12 @@ var fireGlobalEvent = (function() {
         } catch (e) {}
         if (!customEventWorks) {
             try {
-                var event = document.createEvent("CustomEvent");
+                var event;
+                if (document.createEventObject) {
```
Comment (davydog187) [inline src/captured_trace.js]: possibly this would be better split out as a utility function since its used twice? the comment would go nicely there.

**Comment by benjamingr** [comment]
@zomaish any update?

**Comment by zomaish** [comment]
I reached out to Microsoft for an explanation and waiting for a response. I will reach out again and hopefully they can let me know why using document.createEvent causes a leak.

**Comment by petkaantonov** [comment]
I don't see how this would work, `createEventObject` is the old proprietary IE6-8 API that works with `attachEvent`, `fireEvent` and `detachEvent`. `createEvent` is DOM2 api that works with `addEventListener`, `dispatchEvent` and `removeEventListener`,.

**Comment by petkaantonov** [comment]
closing as per above

**Pull request closed without merging**